### DISCUSSION TITLE
Show your workspace!

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2381] [General Discussion]
Share a picture of where you work on your future gaming masterpiece! 

Here's my current setup (for work and Evennia, some gaming etc):

![20210415_160320](https://user-images.githubusercontent.com/294267/114883007-ceb5a280-9e04-11eb-9cdf-c7b841bdb699.jpg)

- Thinkpad p43s (15" screen, meant to be mobile, but with COVID, it has become mostly stationary). Runs Linux Mint Cinnamon with `i3` window manager (minimal mouse need).
- Separate stationary workstation for art/gaming etc (Runs Linux Mint Cinnamon and standard manager)
- Dell Ultrasharp 25" monitor (rotated 90 degrees for coding)
- Dell Ultrasharp 27" monitor 
- Wacom Cintiq 16 (for artwork)
- Das Keyboard mechanical keyboard (Blue Cherry MX switches)
- MouseTrapper mechanical mousepad (instead of moving hand to mouse when needed)
- Voxicon 'Gaming mechanical keyboard' (for art program shortcuts and some gaming)
- Some other stuff, like sound system and printer out of frame
- On a raisable desk (from IKEA of course! ;) )

**Comment by pruuk** [comment]
![PXL_20210426_153407963](https://user-images.githubusercontent.com/72707866/117091147-9f18fc80-ad17-11eb-83f7-d86152683f3f.jpg)
Here is my setup. I still need to build a stand/cabinet for my new desktop, but I haven't felt like doing any woodworking lately. The posters are from the JPL and can be downloaded and printed (like I've done).

**Reply by Griatch** [reply]
A nice, compact setup. Those JPL posters are classic -looks nice there. That's a wiiiide monitor! :open_mouth:

**Comment by samjaninf** [comment]
Oh man, you guys are so organized and neat. Makes me kind of jealous.

**Reply by pruuk** [reply]
I just moved into my house a few months ago. I haven't had a chance to clutter my office yet. Before this, I was living in a school bus and traveling around:
![PXL_20201223_132335306](https://user-images.githubusercontent.com/72707866/117366445-ff29b300-ae7d-11eb-9559-47f027aeb60b.jpg)
The bus was considerably more cluttered inside:
![PXL_20210214_163127142](https://user-images.githubusercontent.com/72707866/117366517-18326400-ae7e-11eb-9b5f-bec02506f1da.jpg)

**Reply by Griatch** [reply]
Heh, that's quite a difference compared to your current workspace's tidyness, indeed. :smile:

**Comment by ghost** [comment]


**Reply by Griatch** [reply]
I like the big bottle of glass cleaner always at hand. I'm sure it's to keep the view out the window as pristine as possible. :stuck_out_tongue_winking_eye:

**Comment by kovitikus** [comment]
@Griatch What's this thing you got between your keyboard and tablet for?
![keypad of some sort](https://i.imgur.com/OePp9gJ.png)

**Reply by Griatch** [reply]
@kovitikus It's technically a 'gaming keyboard' - basically the left side of a regular keyboard with the WASD keys and some of the most common keys used for gaming (the big round button on the bottom left is the space key). 
I mainly use it to group shortcut keys in a central place when drawing though - I can use it with the left hand while drawing with the right (it has a big, confy palm rest too). Also useful since I switch the main keyboard between two computers and may occationally have it switched to another one than the drawing table is linked to.

**Comment by kovitikus** [comment]
![Kovitikus' Workspace](https://i.imgur.com/bj3H9W8.jpg)
***
### Audio Setup

* **Microphone:** [Audio-Technica 2035](https://www.sweetwater.com/store/detail/AT2035--audio-technica-at2035-large-diaphragm-condenser-microphone)

* **Microphone Pop Filter:** [hakan P110](https://www.sweetwater.com/store/detail/hakanP110--hakan-p110-pop-killer-with-gooseneck)

* **Microphone Boom Arm:** [Rode PSA1](https://www.sweetwater.com/store/detail/PSA1Rode--rode-psa1-professional-studio-boom-arm)

* **Desktop Speakers:** [JBL 305P MkII 5-inch Powered Studio Monitor](https://www.sweetwater.com/store/detail/LSR305MK2--jbl-305p-mkii-5-inch-powered-studio-monitor)

* **Desktop Speaker Stands:** [On-Stage Stands SMS4500-P Desktop Monitor Stands](https://www.sweetwater.com/store/detail/SMS4500P--on-stage-stands-sms4500-p-desktop-monitor-stands)

* **Desktop Speaker Stands Stands:** [Amazon Basics Adjustable Computer Monitor Riser Desk Stand](https://www.amazon.com/gp/product/B00X4SCCFG/)

* **Speaker Isolation Pads:** [RAB Audio ProJax 40 Studio Monitor Isolation Kit](https://www.sweetwater.com/store/detail/ProJax40--rab-audio-projax-px40-studio-monitor-isolation-kit-8-pieces)

* **Headphones:** [Sennheiser PC 360 Special Edition Gaming Headset](https://www.amazon.com/gp/product/B01LDTRJG4/)

* **Audio Mixer:** [YAMAHA MG10XU 10-Input Stereo Mixer with Effects](https://www.amazon.com/gp/product/B00IBIVL42/)

***
### Peripherals

* **Mouse:** [Redragon M901 Wired Gaming Mouse MMO RGB LED Backlit Mice 12400 DPI Perdition with 18 Programmable Buttons Weight Tuning for Windows PC Gaming (Black)](https://www.amazon.com/gp/product/B00GU8W5AE/)

* **Mouse Pad:** [Razer Goliathus Medium CONTROL Soft Gaming Mouse Mat](https://www.newegg.com/razer-goliathus-control-edition/p/N82E16826999116)

* **Keyboard:** [Corsair Gaming K95 RGB Mechanical Gaming Keyboard, Cherry MX Brown ](https://www.amazon.com/gp/product/B0149QBRCA/)

* **Keyboard Wrist Support:** [Imak Keyboard Cushion](https://www.amazon.com/gp/product/B001GAOHOI/)

* **Tablet:** [Samsung - Galaxy Tab S7 Plus - 12.4” - 512GB - With S Pen - Wi-Fi - Mystic Black](https://www.bestbuy.com/site/samsung-galaxy-tab-s7-plus-12-4-512gb-with-s-pen-wi-fi-mystic-black/6426498.p?skuId=6426498)

* **Drawing Tablet:** [Huion INSPIROY Q11K Wireless Digital Graphics Drawing Pen Painting Tablet with 8192 Levels of Pressure 8 Express Keys and Pen Holder](https://www.amazon.com/gp/product/B071JN86L3/)

* **Game Controller:** [Microsoft Xbox 360 Wired Controller for Windows](https://www.amazon.com/Microsoft-Wired-Controller-Windows-Console/dp/B004QRKWLA)

* **External Hard Drive x 2:** [WD - easystore 8TB External USB 3.0 Hard Drive - Black](https://www.bestbuy.com/site/wd-easystore-8tb-external-usb-3-0-hard-drive-black/5792401.p?skuId=5792401)

*** 
### PC Parts
* **PC Case:** [Rosewill THOR V2 Gaming ATX Full Tower Computer Case](https://www.newegg.com/p/N82E16811147053)

* **Graphics Card:** [EVGA GeForce GTX 980 Ti 6GB FTW GAMING ACX 2.0+, Whisper Silent Cooling w/ Free Installed Backplate Graphics Card 06G-P4-4996-KR ](https://www.amazon.com/gp/product/B014PXHC6M/)

* **Motherboard:** [ASUS X99-A/USB 3.1 ATX DDR4 3300 (o.c.) Intel LGA 2011](https://www.amazon.com/gp/product/B00VUK55M2/)

* **CPU:** [Intel Core i7-5820K Desktop Processor (6-Cores, 3.3GHz, 15MB Cache, Hyper-Threading Technology)](https://www.amazon.com/gp/product/B00MMLXIKY/)

* **CPU Cooler:** [Cooler Master Hyper 212 Evo CPU Cooler, 4 CDC Heatpipes, 120mm PWM Fan, Aluminum Fins for AMD Ryzen/Intel LG1151 ](https://www.amazon.com/gp/product/B005O65JXI/)

* **RAM:** [Corsair Vengeance LPX 16GB (4x4GB) 2133MHz C13 DDR4 DRAM Memory Kit – Black](https://www.amazon.com/gp/product/B00PLER544/)

* **Power Supply:** [CORSAIR HX Series HX750 750W ATX12V 2.3 / EPS12V 2.91 SLI Ready CrossFire Ready 80 PLUS GOLD Certified Modular Active](https://www.newegg.com/corsair-hx-series-hx750-750w/p/N82E16817139010)

* **Solid State Drive:** [SAMSUNG 840 EVO 2.5" 500GB SATA III TLC Internal Solid State Drive (SSD) MZ-7TE500BW](https://www.newegg.com/samsung-840-evo-500gb/p/N82E16820147249)

* **Hard Disk Drive:** [WD Black 5TB Performance Desktop Hard Disk Drive - 7200 RPM SATA 6Gb/s 128MB Cache 3.5 Inch - WD5001FZWX](https://www.amazon.com/Black-Performance-Desktop-Hard-Drive/dp/B013DHNLN4)

* **Hard Disk Drive 2:** [HGST Deskstar NAS 3.5" 8TB 7200 RPM 128MB Cache SATA 6.0Gb/s](https://www.neweggbusiness.com/product/product.aspx?item=9b-22-146-142)

* **Blu-ray Burner:** [LG Black Blu-ray Burner SATA WH16NS40](https://www.newegg.com/lg-wh16ns40-internal-blu-ray-burner/p/N82E16827136269)

***
### Network
* **Modem:** [TP-Link DOCSIS 3.0 (8x4) High Speed Cable Modem](https://www.newegg.com/tp-link-tc-7610-cable-modem/p/N82E16825165010)

* **Router:** [ASUS Dual-Band 3 x 3 AC1750 Wi-Fi 4-Port Gigabit Router (RT-AC66U_B1)](https://www.amazon.com/gp/product/B01N08LPPP/)

***
### Furniture
* **Floor Mat:** [Best Choice Products 47x59in PVC Home Office Multi-Purpose Chair Floor Mat Hardwood Protector - Clear](https://www.amazon.com/gp/product/B07BSQBTBG/)

* **Chair:** [TECHNI MOBILI 26 in. Width Big and Tall Black Mesh Ergonomic Chair with Adjustable Height](https://www.homedepot.com/p/TECHNI-MOBILI-26-in-Width-Big-and-Tall-Black-Mesh-Ergonomic-Chair-with-Adjustable-Height-RTA-1010-BK/311398457)

* **Desk:** [Bush Business Furniture Series C 72W x 30D Desk Shell in Mocha Cherry ](https://www.amazon.com/gp/product/B07GVT26X5/)

* **Keyboard Tray:** [Bush Business Furniture Universal Keyboard Shelf with Galaxy Finish](https://www.bushfurniturecollection.com/Bush-AC99808-Keyboard-Shelf.htm)

**Reply by Griatch** [reply]
A nice setup! You forgot to list the brand of your pair of scissors though. :tongue:

**Reply by kovitikus** [reply]
> 
> 
> A nice setup! You forgot to list the brand of your pair of scissors though. 👅

Funny you mention that. I love them and I was considering listing them for fun.

I don't have a link, since I bought them randomly at Walmart and I haven't seen this exact model there again, but they are Fiskars. I would never buy any other brand of scissors. They are the best.

Ok I lied, I found a link on the manufacturer's site. 

https://www.fiskars.com/en-us/crafting-and-sewing/products/scissors-and-shears/stitcher-scissors-no-5-01-005435

**Reply by Griatch** [reply]
Fiskars is a Scandinavian brand (Finnish) so you have good taste in scissors, I'll give you that. :slightly_smiling_face:

**Reply by kovitikus** [reply]
The pen is [Uni-Ball Classic](https://www.walmart.com/ip/Uni-Ball-Classic-Rollerball-Pens-1-Dozen-Quantity/15066665). Their pens are reliable and competitively priced. I bought a box of them years ago and never had one die on me prematurely.

I guess the fact that they are a Japanese company explains their reliability.
> Uni-ball and Uni are brands of pens and pencils, made by the Mitsubishi Pencil Company Limited (三菱鉛筆株式会社, Mitsubishi Enpitsu Kabushiki Gaisha) of Japan. The brand was introduced in 1979 as a rollerball pen model, then expanding to the rest of Mitsubishi Pencil products.

https://en.wikipedia.org/wiki/Uni-ball

**Comment by Sk33t3rson** [comment]
Oooo what fun, here the home of Aagonar, soon to change need to one monitor to vertical because why not!
![a165d2d3-e180-43b8-904f-cec62d264a5e](https://user-images.githubusercontent.com/88866796/202487907-3dd70714-61dd-4b59-b030-6fcb918bfa72.jpg)

**Reply by Griatch** [reply]
I like that raised monitor in the center! Yeah, having a vertically rotated monitor is very nice for coding.

**Comment by iLPdev** [comment]
The initial incarnation of the _taviMUD lab_ in my business space... pending a move of the desk an monitors into the smaller albeit vacant office on the other side of my waiting room.
![image](https://github.com/evennia/evennia/assets/1428343/9aa543ed-ce1c-4fbd-86ed-32f7fecf6a2f)

![image](https://github.com/evennia/evennia/assets/1428343/423cd5a1-1e59-476a-b950-ae4f6e4626b0)

**Comment by KathyWithAK** [comment]
![IMG_20231025_210719](https://github.com/evennia/evennia/assets/37349651/057875e8-326f-48a6-b5dd-c3544cb7684b)

My work area has gotten a bit antisocial, but at least I have lots of toys to keep me from going insane with boredom during tele-meetings.
